bin/menmosd/src/menmosd/server/server_impl.rs

```diff
@@ -23,6 +23,10 @@ impl Server {
         node: N,
     ) -> Result<Server> {
         let node: Arc<Box<dyn DirectoryNode + Send + Sync>> = Arc::new(Box::new(node));
+
+        // Create the admin user.
+        node.register("admin", &cfg.node.admin_password).await?;
```

Comment:
What if the admin was already registered? I mean if you restart menmosd, this function will be called twice.

Reply:
The user will be overwritten - simple as that. It's the behavior we want because it will overwrite the account if the admin password changed in the config.